CMakeLists.txt

```diff
@@ -347,7 +347,8 @@ find_package(SWIG)
 
 if(SWIG_FOUND)
     set(SWIG_VERSION_CHECK FALSE)
-    if("${SWIG_VERSION}" VERSION_GREATER ${GR_SWIG_MIN_VERSION})
+    if("${SWIG_VERSION}" VERSION_EQUAL ${GR_SWIG_MIN_VERSION} OR
+       "${SWIG_VERSION}" VERSION_GREATER ${GR_SWIG_MIN_VERSION})
```

Comment:
Yeah, master is at 2.0.3, I'll change it to 2.0.4.  Its more clear as documentation to make this the actual version instead of off by one.  I myself would have forgotten then after only a few days...